marketplace/deployer_util/config_helper.py

```diff
@@ -168,6 +168,14 @@ def validate(self):
               'x-google-marketplace.images')
       if self._x_google_marketplace._deployer_service_account:
         self._x_google_marketplace._deployer_service_account.validate()
+        # Move to validate() once enforced on SERVICE_ACCOUNT properties as well.
+        if self._x_google_marketplace._deployer_service_account.has_discouraged_cluster_scoped_permissions(
```

Comment:
can be split into multiple lines with the following trick:
```python
if (self._x_google_marketplace
    ._deployer_service_account
    ._has_discouraged_cluster_scoped_permissions()):
```

Reply:
Ah, much better - I forgot about that one since not added by our autoformatter.